### DISCUSSION TITLE
Where does Authenticator save your accounts on Windows?

### DISCUSSION THREAD

**Discussion by jc6github** [Authenticator-Extension/Authenticator#833] [Q&A] [answered]
After you add the Authenticator Extension in Chrome and add an account with QR code, where does the account data get saved to? (assuming you don't do a backup at all)
I'm assuming someplace in user's appdata folder?
If someone can point me to exact folder/file location it would be appreciated :)

**Comment by Sneezry** [comment, marked as answer]
In Windows, if you enabled browser sync, it is saved `%LocalAppData%\Google\Chrome\User Data\Default\Sync Extension Settings\bhghoamapcdpbohphigoooaddinpkbai`. If you disabled browser sync, it is saved `%LocalAppData%\Google\Chrome\User Data\Default\Local Extension Settings\bhghoamapcdpbohphigoooaddinpkbai`.

**Reply by jc6github** [reply]
Thanks!  Also, is it automatically encrypted, or is that set by the account that's scanned?

**Reply by Sneezry** [reply]
You can set a master password to encrypt the data. To set a master password, go to Security menu.
